roles/ceph-facts/tasks/grafana.yml

```diff
@@ -21,6 +23,7 @@
     grafana_server_addrs: "{{ (grafana_server_addrs | default([]) + [hostvars[item]['ansible_all_ipv4_addresses'] | ips_in_ranges(public_network.split(',')) | first]) | unique }}"
   with_items: "{{ groups.get(grafana_server_group_name, []) }}"
   when:
+    - grafana_server_addr is not defined
```

Comment:
if you skip this task when `grafana_server_addr` is defined by the user, how `grafana_server_addrs` is supposed to be set ? It is consumed

Reply:
Sorry, the original patch was over cae24dd85a979ed40efc7de384c158a04bd3558d and I messed it up when rebasing before sending this PR. The new patch I'm force pushing will employ a new approach.